controller/model/edge_router_manager.go

```diff
@@ -232,6 +234,16 @@ func (self *EdgeRouterManager) IsAccessToEdgeRouterAllowed(identityId, serviceId
 	return result, nil
 }
 
+func (self *EdgeRouterManager) ListForIdentityAndServiceWithTx(tx *bbolt.Tx, identityId, serviceId string) (*EdgeRouterListResult, error) {
+	query := fmt.Sprintf(`anyOf(identities) = "%v" and anyOf(services) = "%v" SORT BY connected DESC LIMIT 25`, identityId, serviceId)
```

Comment:
still feels like a moving target. feels like 25 shouldn't be hardcoded in this way. also feels like maybe this should be randomized somehow?

Reply:
I agree that this should be configurable in some way. Most flexible would probably be per service or per identity. This fix will solve the immediate issue, and hopefully we can make it more flexible with some follow-up work.